tests/testthat/setup-testing_depth.R

```diff
@@ -0,0 +1,56 @@
+#' Returns testing depth set by an environmental variable.
+#'
+#' @details Looks for the option `TESTING_DEPTH` first, if not set,
+#' takes the system environmental variable `TESTING_DEPTH`. If neither
+#' is set, then returns 3 by default. If the value of `TESTING_DEPTH`
+#' is not a scalar numeric, then returns 3.
+#'
+#' @return `numeric(1)` the testing depth.
+#'
+testing_depth <- function() {
```

Comment:
Can we get the function name to be called `testing_depth` but the variable inside be called `depth`?

Reply:
Sure! Nice consistancy.

Reply:
Done in 75fa708792334ef552b35417c3fdb1fe14587536